src/app/gateways/views/BuiltinGatewayDetailView.vue

```diff
@@ -1,116 +1,256 @@
+                :page-number="route.params.page"
+                :page-size="route.params.size"
+                :headers="[
+                  { label: 'Name', key: 'name' },
+                  ...(can('use zones') ? [{ label: 'Zone', key: 'zone' }] : []),
+                  { label: 'Certificate Info', key: 'certificate' },
+                  { label: 'Status', key: 'status' },
+                  { label: 'Warnings', key: 'warnings', hideLabel: true },
+                  { label: 'Details', key: 'details', hideLabel: true },
+                ]"
+                :items="dataplanesData?.items"
+                :total="dataplanesData?.total"
+                :error="dataplanesError"
+                :is-selected-row="(row) => row.name === route.params.dataPlane"
+                summary-route-name="builtin-gateway-data-plane-summary-view"
+                :empty-state-message="t('common.emptyState.message', { type: 'Data Plane Proxies' })"
+                :empty-state-cta-to="t('data-planes.href.docs.data_plane_proxy')"
+                :empty-state-cta-text="t('common.documentation')"
+                @change="route.update"
               >
-                <ErrorBlock
-                  v-if="dataplanesError"
-                  :error="dataplanesError"
-                />
-
-                <LoadingBlock v-else-if="dataplanesData === undefined" />
-
-                <EmptyBlock v-else-if="dataplanesData.items.length === 0" />
-
-                <template v-else>
-                  <ul data-testid="affected-data-plane-proxies">
-                    <li
-                      v-for="(policyDataplane, key) in dataplanesData.items.filter((policyDataplane) => policyDataplane.name.toLowerCase().includes(route.params.dataplane.toLowerCase()))"
-                      :key="key"
-                      data-testid="dataplane-name"
-                    >
-                      <RouterLink
-                        :to="{
-                          name: 'data-plane-detail-view',
-                          params: {
-                            mesh: policyDataplane.mesh,
-                            dataPlane: policyDataplane.name,
-                          },
-                        }"
-                      >
-                        {{ policyDataplane.name }}
-                      </RouterLink>
-                    </li>
-                  </ul>
+                <template #toolbar>
+                  <FilterBar
+                    class="data-plane-proxy-filter"
+                    :placeholder="`tag: 'kuma.io/protocol: http'`"
+                    :query="route.params.query"
+                    :fields="{
+                      name: { description: 'filter by name or parts of a name' },
+                      protocol: { description: 'filter by “kuma.io/protocol” value' },
+                      tag: { description: 'filter by tags (e.g. “tag: version:2”)' },
+                      zone: { description: 'filter by “kuma.io/zone” value' },
```

Comment:
Shall we use `can('use zones')` here seeing as we are hiding the column?

Reply:
Actually if we want to do that, lets do that as a follow up everywhere instead of just here.

Reply:
👍🏻